Plugins/Creature/NWScript/nwnx_creature.nss

```diff
@@ -1092,44 +1120,22 @@ void NWNX_Creature_SetAlignmentLawChaos(object creature, int value)
 
 int NWNX_Creature_GetClericDomain(object creature, int index)
```

Comment:
`WriteTimestampedLogEntry("WARNING: NWNX_Creature_GetClericDomain() is deprecated, please use NWNX_Creature_GetDomain()");` here too please!